metagraph/src/graph/alignment/aligner_extender_methods.cpp

```diff
@@ -32,9 +33,7 @@ ::initialize_query(const std::string_view query) {
     assert(config_.match_score(query) == partial_sums_.front());
     assert(config_.get_row(query.back())[query.back()] == partial_sums_.back());
 
-    profile_score.clear();
-    profile_op.clear();
```

Comment:
hm, was this not used? Or now you assume you create a new DefaultColumnExtender for each alignment?

Maybe add `assert(profile_score.size() == 0);`?

Reply:
Before, I used to create an `Extender` first, then `initialize` it with a query. I've now combined those two so the query is passed to the constructor. Since it's in the constructor, we know that they're empty already.